app/patient-details/views/patient-arv-pickup-history-report.html

```diff
@@ -3,97 +3,111 @@
+            <td ng-repeat="x in calendar[3] track by $index">{{x}}</td>
           </tr>
-          </tbody>
         </table>
       </div>
-    </div>
   </div>
 
-  <div class="row">
-    <div class="col-xs-10 col-xs-offset-1">
-      <h2>Levantamentos</h2>
-      <div class="pickups">
-        <table class="table table-bordered table-striped">
-          <thead>
-            <tr>
-              <th>{{'PHARMACY_PICKUP_DATE' | translate}}</th>
-              <th>{{'PHARMACY_REGIMEN'     | translate}}</th>
-              <th>{{'PHARMACY_QUANTITY_SHORT' | translate}}</th>
-              <th>{{'PHARMACY_POSOLOGY'    | translate}}</th>
-              <th>{{'PHARMACY_NEXT_PICKUP' | translate}}</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr ng-repeat="p in patient.pickups">
-              <td>{{p.encounterDatetime | date}}</td>
-              <td>{{p.regimen}}</td>
-              <td>{{p.quantity}}</td>
-              <td>{{p.posology}}</td>
-              <td>{{p.nextPickup  | date}}</td>
-            </tr>
-          </tbody>
-        </table>
+  <div id="top-right-area">
+    <h2 id="top-right-area-title">{{'FILA_REPORT_PERSONAL_DATA_NOTE' | translate}}</h2>
+    <div id="patient-data-container">
+      <div style="font-size: 8pt; font-weight: bold; width: 40mm; margin-left: 3mm; align-self: center; font-size: 10pt; text-align: center;">
+        <span id="nid-label">{{'SEARCH_PATIENT_NID' | translate}}:</span> {{patient.identifier}}
+      </div>
+      <div id="personal-data-conteiner">
+        <div id="first-line-personal-data" class="personal-data">Nº do Livro TARV: ______, Pág: _____, Linha: _____</div>
+        <div class="personal-data">{{'SEARCH_PATIENT_FIRSTNAME' | translate}}: {{patient.fullName}}</div>
+        <div class="personal-data">{{'PATIENT_INFO_OTHERS' | translate}}: {{patient['Numero de Telefone 1']}} / {{patient['Numero de Telefone 2']}}</div>
+        <div class="personal-data">{{'PATIENT_INFO_ADDRESS' | translate}}: {{patient.address.display}}</div>
       </div>
     </div>
   </div>
+
+</div>
+
+
+<div id="main-container">
+
+  <table style="width: 145mm;">
+      <thead>
+        <tr>
+          <th>{{'DISPENSATION_DATE' | translate}}</th>
+          <th class="medicine-column">Medicamentos ARVs</th>
+          <th>Quantidade Aviada</th>
+          <th class="dosage-column">Dosagem</th>
+          <th>Data do Próximo Levantamento</th>
+        </tr>
+      </thead>
+      <tbody >
+        <tr ng-repeat="displacement in filaDisplacements1 track by $index" class="displacement-row">
+          <td ng-if="displacement.dateRowSpanSize > 0" rowspan="{{displacement.dateRowSpanSize}}">{{displacement.date | date:"dd/MM/yyyy"}}&nbsp;</td>
+          <td>{{displacement.medicine}}&nbsp;</td>
+          <td>{{displacement.quantity}}&nbsp;</td>
+          <td>{{displacement.dosage}}&nbsp;</td>
+          <td>{{displacement.nextDisplacement | date:"dd/MM/yyyy"}}&nbsp;</td>
```

Comment:
Date format should not be hard-coded. For this case ```{{displacement.nextDisplacement | date}}``` is sufficient. See: https://github.com/esaude/esaude-emr-poc/blob/master/app/common/application/constants.js and ```dateFilterDecorator``` on https://github.com/esaude/esaude-emr-poc/blob/master/app/common/application/config.js.